packages/contracts-bedrock/periphery-deploy-config/sepolia.json

```diff
@@ -1,42 +1,69 @@
 {
-  "faucetAdmin": "0x212E789D4523D4BAF464f8Fb2A9B9dff2B36e5A6",
+  "create2DeploymentSalt": "0.0.2",
+
+  "gelatoAutomateContract": "0x2A6C106ae13B558BB9E2Ec64Bd2f1f7BEFF3A5E0",
+  "gelatoTreasuryContract": "0x7506C12a824d73D9b08564d5Afc22c949434755e",
+
+  "operationsDrippieOwner": "0x03C256F7Ae7518D0fe489F257ab4b928D37CBE16",
+  "operationsSequencerDripV1Target": "0x8F23BB38F531600e5d8FDDaAEC41F13FaB46E98c",
+  "operationsSequencerDripV1Value": 20000000000000000000,
+  "operationsSequencerDripV1Interval": 86400,
+  "operationsSequencerDripV1Threshold": 100000000000000000000,
+  "operationsGelatoDripV1Recipient": "0x03C256F7Ae7518D0fe489F257ab4b928D37CBE16",
+  "operationsGelatoDripV1Value": 1000000000000000000,
+  "operationsGelatoDripV1Interval": 86400,
+  "operationsGelatoDripV1Threshold": 500000000000000000,
+  "operationsSecretsDripV1Delay": 43200,
+  "operationsSecretsDripV1MustExist": "0x565fa8c7daa859353b5b328b97b12c7d66c5832b2a24d4e0f739a65ad266a46f",
+  "operationsSecretsDripV1MustNotExist": "0xbc362b01d69a85dff1793803dde67df1f338f37a36fdc73dddf27283d215e614",
+  "operationsSecretsDripV1Target": "0x03C256F7Ae7518D0fe489F257ab4b928D37CBE16",
+  "operationsSecretsDripV1Value": 1000000000000000000,
+  "operationsSecretsDripV1Interval": 3600,
```

Comment:
Any references for this new configuration? Values seem sane, curious if these were discussed somewhere.

Reply:
Addressing this later after next PR goes thru.